src/upload.js

```diff
@@ -37,20 +50,21 @@ function UploadDirectory(path, opts) {
     formData.append(opts.portalDirectoryFileFieldname, fs.createReadStream(file), { filepath: file });
   }
 
+  // Form the URL.
   const url = `${trimTrailingSlash(opts.portalUrl)}${trimTrailingSlash(opts.portalUploadPath)}?filename=${
     opts.customFilename || path
   }`;
 
   return new Promise((resolve, reject) => {
     axios
       .post(url, formData, { headers: formData.getHeaders() })
-      .then((resp) => {
-        resolve(`sia://${resp.data.skylink}`);
+      .then((response) => {
+        resolve(`${UriSkynetPrefix}${response.data.skylink}`);
       })
       .catch((error) => {
         reject(error);
       });
   });
 }
 
-module.exports = { UploadFile, UploadDirectory };
+module.exports = { DefaultUploadOptions, UploadFile, UploadDirectory };
```

Comment:
same as with `DefaultDownloadOptions`, we don't need to expose `DefaultUploadOptions` since they will be merged in with the `customOptions` anyway